Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -567,6 +566,28 @@ static void addAttributesForFontPalettes(CFMutableDictionaryRef attributes, cons
     }
 }
 
+static std::optional<bool>& overrideEnhanceTextLegibilityStorage()
+{
+    static NeverDestroyed<std::optional<bool>> overrideEnhanceTextLegibility;
+    return overrideEnhanceTextLegibility.get();
+}
+
+void overrideEnhanceTextLegibility(bool override)
```

Comment:
I'm surprised the compiler let you use a keyword as a variable name.

Reply:
C++ parsing is context-aware 😅